deploy/clowdapp.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ingress
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ingress
+  spec:
+    envName: ${ENV_NAME}
+    pods:
+    - name: ingress
+      image: ${IMAGE}:${IMAGE_TAG}
+      minReplicas: ${{REPLICAS}}
+      livenessProbe:
+        failureThreshold: 3
+        httpGet:
+          path: /api/ingress/v1/version
+          port: 8000
+          scheme: HTTP
+        initialDelaySeconds: 5
+        periodSeconds: 5
+        successThreshold: 1
+        timeoutSeconds: 120
+      readinessProbe:
+        failureThreshold: 3
+        httpGet:
+          path: /api/ingress/v1/version
+          port: 8000
```

Comment:
Same question as above. If we're running on 3000, then the readiness/liveliness should check 3000 as well, right?